Libplanet/Net/Swarm.cs

```diff
@@ -499,14 +445,59 @@ private set
             }
         }
 
+        public async Task BootstrapAsync(
+           IEnumerable<Peer> seedPeers,
+           double pingSeedTimeout,
+           double findPeerTimeout,
+           CancellationToken cancellationToken = default(CancellationToken))
+        {
+            await BootstrapAsync(
+                seedPeers,
+                TimeSpan.FromMilliseconds(pingSeedTimeout),
+                TimeSpan.FromMilliseconds(findPeerTimeout),
+                cancellationToken);
+        }
+
+        /// <summary>
+        /// Join to the peer-to-peer network using seed peers.
+        /// </summary>
+        /// <param name="seedPeers">List of seed peers.</param>
+        /// <param name="pingSeedTimeout">Timeout for connecting to seed peers.</param>
+        /// <param name="findPeerTimeout">Timeout for requesting neighbours.</param>
+        /// <param name="cancellationToken">A cancellation token used to propagate notification
+        /// that this operation should be canceled.</param>
+        /// <returns>An awaitable task without value.</returns>
+        /// <exception cref="SwarmException">Thrown when this <see cref="Swarm{T}"/> instance is
+        /// not <see cref="Running"/>.</exception>
+        public async Task BootstrapAsync(
+            IEnumerable<Peer> seedPeers,
+            TimeSpan? pingSeedTimeout,
+            TimeSpan? findPeerTimeout,
+            CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (cancellationToken == default(CancellationToken))
+            {
+                cancellationToken = _cancellationToken;
```

Comment:
If user calls this method without cancellation token, then uses the cancellation token given in PrepareAsync so that BootstrapAsync can be canceled when StopAsync is called.